share/spice/goodreads/goodreads.js

```diff
@@ -86,7 +86,7 @@
                     brand: true,
                     moreAt: true,
                     ratingtext: true,
-                    subtitle_content: Spice.goodreads.details
+                    buy: Spice.goodreads.details
                 },
```

Comment:
This is done so that the description comes *before* the **More Info** table and the rating. This feels way natural that all essential data is present before an optional **More Info**.